src/components/select/__test__/select.spec.js

```diff
@@ -1,34 +1,64 @@
 import Select from '../'
 
 it('Test <Select /> default `foo`', ()=> {
-  const vm = render((h)=> {
-    return (
-      <Select />
-    )
+  const vm = shallow({
+    render(h) {
+      return (
+        <Select />
+      )
+    }
   })
 
   expect(vm.foo).toBe(1)
 })
 
-it('Test <Select /> `handleClick`', ()=> {
-  const vm = render((h)=> {
-    return (
-      <Select />
-    )
+it('Test <Select /> `handleClick`', async()=> {
+  const vm = shallow({
+    render(h) {
+      return (
+        <Select />
+      )
+    }
   })
 
   vm.handleClick()
+  await new Promise((done)=> {
+    vm.$nextTick(done)
```

Comment:
`In order to wait until Vue.js has finished updating the DOM after a data change.`
http://vuejs.org/guide/reactivity.html#Async-Update-Queue